LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/BasicsStation/CupsTwinInfo.cs

```diff
@@ -51,5 +57,17 @@ internal sealed record CupsTwinInfo
         [JsonPropertyName("tcCredentialUrl")]
         [JsonProperty("tcCredentialUrl")]
         public string TcCredentialUrl { get; }
+
+        [JsonPropertyName("package")]
+        [JsonProperty("package")]
+        public string Package { get; }
+
+        [JsonPropertyName("fwKeyChecksum")]
+        [JsonProperty("fwKeyChecksum")]
+        public int FwKeyChecksum { get; }
+
+        [JsonPropertyName("fwSignature")]
+        [JsonProperty("fwSignature")]
+        public string FwSignature { get; }
```

Comment:
What about calling this `FwSignatureInBase64` so it's obvious the string is Base64-encoded data?